### DISCUSSION TITLE
Rigid Body x and y-constraints not set correctly in GOM file

### DISCUSSION THREAD

**Discussion by WaveHello** [Anura3D/Anura3D_OpenSource#130] [General]
Hello All,

In Anura3D_2023, when using a Rigid Body formulation, the x and/or y-constraint flag isn't correctly set in the .GOM file. The x and y constraint flag is always zero. 

See the below screenshot showing the x-constraint flag set to "Yes" in GiD:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/110841896/8911c483-9a88-4ee1-a88f-4681735e9d3a)

The below screenshot shows the x-constraint in the .GOM file generated from the above GiD file:
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/110841896/d60dd61f-f443-4c08-8975-37d11ca48774)

I can manually set the constraint in the .GOM file and the model runs perfectly.

I'm using GiD version 16.0.5 and the latest release of Anura3D_2023. To ease the process of recreating the problem I've attached the GiD file for the Impact problem (The 2D Plane-Strain variation with a Rigid Body for the Impacting Body) in the tutorial manual and the generated .GOM and .CPS files. 

[Rigid_Constraint_Flag_Issue.zip](https://github.com/Anura3D/Anura3D_OpenSource/files/12494280/Rigid_Constraint_Flag_Issue.zip)

**Comment by gdicarluccio** [comment]
Thanks for your feedback! The Problem Type has been just updated to fix this issue.

**Reply by WaveHello** [reply]
Thank you!
